miso-web/src/main/webapp/scripts/form_submission.js

```diff
@@ -0,0 +1,68 @@
+if (typeof FormTarget === 'undefined') {
+  FormTarget = {};
+}
+FormTarget.submission = (function($) {
+
+  return {
+    getSaveUrl: function(submission) {
+      return submission.id ? ('/miso/rest/submissions/' + submission.id) : '/miso/rest/submissions';
+    },
+    getSaveMethod: function(submission) {
+      return submission.id ? 'PUT' : 'POST';
+    },
+    getEditUrl: function(submission) {
+      return '/miso/submission/' + submission.id;
+    },
+    getSections: function(config, object) {
+      return [{
+        title: 'Submission Information',
+        fields: [{
+          title: 'Submission ID',
+          data: 'id',
+          type: 'read-only',
+          getDisplayValue: function(submission) {
+            return submission.id || 'Unsaved';
+          }
+        }, {
+          title: 'Title',
+          data: 'title',
+          type: 'text',
+          required: true,
+          maxLength: 255
+        }, {
+          title: 'Alias',
+          data: 'alias',
+          type: 'text',
+          required: true,
+          maxLength: 100
+        }, {
+          title: 'Description',
+          data: 'description',
+          type: 'text',
+          maxLength: 255
+        }, {
+          title: 'Accession',
+          data: 'accession',
+          type: 'read-only',
+          getLink: function(sample) {
+            return 'http://www.ebi.ac.uk/ena/data/view/' + sample.accession;
```

Comment:
MISO was originally designed for the archive to be swappable with another one. I don't think there are any others in existence in the real world and I ripped out most the code that, so, I think it's fine to hard code these URLs. If you see anything that relates to a swappable archive, I say, rip it out.

Reply:
There's also no way of setting the accession on anything via the UI. This is how it was before too though.